tests/test_ioc_scan.py

```diff
@@ -111,7 +116,57 @@ def test_scan_stdin(capsys):
     ), "standard out should contain eicar detection with filename"
     assert (
         captured.out.count(test_hash) == 2
-    ), "standard out should detection and tally should match hash"
+    ), "standard out detection and tally should match hash"
+    assert (
+        captured.out.count(f"{test_hash}    1") == 1
+    ), "standard out should show one detected match for the test file"
+
+
+@pytest.fixture
+def test_fs(fs):
+    """Set up the fake filesystem for testing standalone mode."""
+    fs.add_real_directory("tests/targets")
+    fs.add_real_file("tests/testblob.txt")
+    yield fs
+
+
+def test_ioc_scanner_standalone_no_file(capsys, test_fs):
+    """Test running the scanner in standalone mode."""
+    test_hash = "69630e4574ec6798239b091cda43dca0"
+
+    with patch.object(
+        sys, "argv", ["bogus"],
+    ):
+        ioc_scanner.main()
+    captured = capsys.readouterr()
+    print(captured.out)
+    assert (
+        captured.out.count("eicar.txt") == 1
+    ), "standard out should contain eicar detection with filename"
+    assert (
+        captured.out.count(test_hash) == 2
+    ), "standard out detection and tally should match hash"
+    assert (
+        captured.out.count(f"{test_hash}    1") == 1
+    ), "standard out should show one detected match for the test file"
+
+
+def test_ioc_scanner_standalone_file(capsys, test_fs):
+    """Test running the scanner in standalone mode with an input target file."""
+    test_hash = "69630e4574ec6798239b091cda43dca0"
```

Comment:
A small improvement here would be to change this `test_hash` value to illustrate that the detected IOC is in fact coming from `testblob.txt` and not from `test_hash`.

Reply:
Just so I'm on the same page, do you mean the value or the variable name?

Reply:
I'm talking about changing the value of the `test_hash`, since it's the same as the EICAR test hash in `testblob.txt` that is being detected.

Reply:
`test_hash` is being used to verify the output of the tool. It needs to be the same value as what's in `testblob.txt` because that's the hash that should be detected when scanning:
https://github.com/cisagov/ioc-scanner/blob/ce0a0b8ceb5abf424723138288824ad278b4b2da/tests/test_ioc_scan.py#L167-L172

Reply:
The variable name could (should?) be changed to more clearly represent how it's being used.

Reply:
Ah- you are right, I am wrong.  What I'm looking for here is a way to show that the IOCs we're searching for are actually coming from the input file, rather than from the [defaults here](https://github.com/cisagov/ioc-scanner/blob/improvement/add_file_option_to_stand-alone_mode/src/ioc_scan/ioc_scanner.py#L23L50).

Reply:
That makes sense and would be a good addition.